packages/core/src/components/results/ChoiceAnswerResult.tsx

```diff
@@ -0,0 +1,118 @@
+import classNames from 'classnames'
+import React, { useContext } from 'react'
+import { connect } from 'react-redux'
+import { getNumericAttribute, mapChildElements, query } from '../../dom-utils'
+import { AppState } from '../../store'
+import { ChoiceAnswer as ChoiceAnswerT, ExamComponentProps, QuestionId } from '../types'
+import { ExamResultsContext, findMultiChoiceFromGradingStructure } from './ExamResultsContext'
+
+interface ChoiceAnswerOptionProps extends ExamComponentProps {
+  questionId: QuestionId
+  selected: boolean
+  direction: string
+  score: number
+  isCorrect: boolean
+}
+
+function ChoiceAnswerOption({
```

Comment:
Tämä komponentti on käytännössä on copy-pastettu suoraan ChoiceAnswerista. Parempi olisi jos ne jakaisivat yhteisen koodin, eli käytännössä varmaankin sama koodi rendaisi `e:choice-answerin` kun taas `e:choice-answer-optioneille` olisi kokeessa ja skopissa eri rendauskoodi.

Reply:
Yritin miettiä vähän tätä, ovat kyllä pitkälti copy-pastea joo, mutta toisaalta pieniä eroavaisuuksia on siellä täällä, niin en ainakaan tältä istumalta nyt keksinyt miten ne saisi yhdistettyä niin, ettei yhdistelykoodista tulisi liian kludgeisaa. Jättäisin tän nyt näin.